server/service/client.go

```diff
@@ -1256,6 +1256,10 @@ func (c *Client) DoGitOps(
 			integrations = map[string]interface{}{}
 			group.AppConfig.(map[string]interface{})["integrations"] = integrations
 		}
+		integrations, ok = integrations.(map[string]interface{})
+		if !ok {
+			return nil, errors.New("org_settings.integrations config is not a map")
+		}
```

Comment:
I see that this required some changes in `DoGitOps`, will that new config also work for `fleetctl apply`? AFAIK we still support it and add the new settings to both, but I could be wrong.

Reply:
Yes, the new config works with `fleetctl apply` -- I manually tested it.